pkg/apis/serving/v1alpha1/configuration_validation_test.go

```diff
@@ -205,13 +209,14 @@ func TestConfigurationSpecValidation(t *testing.T) {
 			Template: &RevisionTemplateSpec{
 				Spec: RevisionSpec{
 					DeprecatedConcurrencyModel: "Multi",
-					Container: &corev1.Container{
+					DeprecatedContainer: &corev1.Container{
 						Image: "hellworld",
 					},
 				},
 			},
 		},
-		want: apis.ErrDisallowedFields("template.spec.concurrencyModel"),
+		want: apis.ErrDisallowedFields(
+			"template.spec.concurrencyModel", "template.spec.container"),
```

Comment:
I thought that deprecated fields will be accepted until a later milestone? 
https://docs.google.com/presentation/d/1mOhnhy8kA4-K9Necct-NeIwysxze_FUule-8u5ZHmwA/edit#slide=id.g55ca79ba88_0_58

Reply:
This is a context-sensitive rejection.  Within the context of the new fields (here `template:`) we will never accept deprecated fields.